docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 5
+title: Tokens Overview
+description: Choose between the Stellar Asset Contract and Custom Tokens
+---
+
+Tokens exist in two forms on Stellar: 1) assets issued by Stellar accounts (Stellar assets can have a [Stellar Asset Contract (SAC)](./stellar-asset-contract.mdx) implementation), and 2) as a [custom token](./token-interface.mdx) created by a smart contract.
+
+Several factors (such as project-specific requirements and network capabilities) can help you determine whether to issue an asset on Stellar or create a custom token with a smart contract for your project.
```

Comment:
There's a lot of brackets in the overview, and the contents of these brackets while useful are not critical. Suggest:
```suggestion
Several factors can help you determine whether to issue an asset on Stellar or create a custom token with a smart contract for your project.
```